src/basic/missing_syscall_def.h

```diff
@@ -61,8 +64,12 @@
 #    endif
 #  elif defined(__powerpc__)
 #    define systemd_NR_bpf 361
-#  elif defined(__riscv) && defined(__LP64__)
-#    define systemd_NR_bpf 280
+#  elif defined(__riscv)
+#    if defined(__LP64__)
```

Comment:
once checked for `defined(__riscv)`
perhaps use  `__riscv_xlen == 64` to denote LP64 and `__riscv_xlen == 32` to denore ILP32